src/components/Map.vue

```diff
@@ -42,34 +42,42 @@ export default {
         minZoom: 1,
         maxZoom: 19
       }).addTo(this.map);
-      var markers = []
-      var huntermtmarker = L.marker(this.huntermtll).addTo(this.map).bindPopup('Hunter Mt')
-      markers.push(huntermtmarker)
-        //.openPopup();
-      
-      L.marker(this.mtcreekll).addTo(this.map)
-        .bindPopup('Mt Creek')
-        //.openPopup();
-      
-      L.marker(this.camalbackll).addTo(this.map)
-        .bindPopup('Camalback')
-        //.openPopup();
+      var huntermtmarker = L.marker(this.huntermtll,{title:"marker_hunter"}).addTo(this.map).bindPopup('Hunter Mt')
```

Comment:
change `var` -> `let` for the whole file.
https://medium.com/@realdennis/%E9%96%92%E8%81%8A-var%E8%88%87let%E7%9A%84%E6%87%B6%E4%BA%BA%E5%8C%85-javascript-b5a3f40ee28d

Reply:
done